frontend/src/old-pages/CustomImages/CustomImageDetails.tsx

```diff
@@ -66,6 +65,19 @@ function CustomImageConfiguration() {
 function CustomImageProperties() {
   const selected = useState([...customImagesPath, 'selected'])
   const image = useState(['customImages', 'index', selected])
+
+  const copyImageConfigUrl = useCallback(
+    () => {
+      navigator.clipboard.writeText(image.imageConfiguration.url);
+    }, [navigator]
```

Comment:
nitpick: in the array of dependencies you should put `[image.imageConfiguration.url)]` instead of `[navigator]` because the function has to be recreated when that variable changes 
navigator is global and does not change

same below

Reply:
Good catch :+1: